### DISCUSSION TITLE
Macbook pro Mid 2010 with OS of slighly older version

### DISCUSSION THREAD

**Discussion by msnijd** [julian-poidevin/MBPMid2010_GPUFix#149] [Q&A]
Good evening,

My macbook pro 2010 has the GNU KERNEL PANIC crashes also.
I am interested in trying the MBPMid2010_GPUFix.
My macbook currently has OS version 10.9.5 installed.
According to the READ.ME the fix will not work with my macbook.
What would  you suggest to do in the particular case ? Is there an older version that works with my OS version ?
Or would I have to upgrade to a newer OS version ?

Best regards and thanks,
Martin

**Comment by julian-poidevin** [comment]
Hello, sorry for the delay. I would suggest updating to 10.10.
